src/plone/restapi/services/linkintegrity/post.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+from plone.app.linkintegrity.utils import linkintegrity_enabled
+from plone.restapi.deserializer import json_body
+from plone.restapi.serializer.converters import json_compatible
+from plone.restapi.services import Service
+from zExceptions import BadRequest
+from zope.interface import implementer
+from zope.publisher.interfaces import IPublishTraverse
+from plone.app.uuid.utils import uuidToObject
+from plone.restapi.interfaces import ISerializeToJsonSummary
+from zope.component import getMultiAdapter
+
+
+@implementer(IPublishTraverse)
+class LinkintegrityPOST(Service):
+    """
+    Return a list of breaches from p.a.linkintegrity view
+    """
+
+    def reply(self):
+        query = json_body(self.request)
+        uids = query.get("uids", [])
+
+        if not uids:
+            raise BadRequest('Missing parameter "uids"')
+
+        if not linkintegrity_enabled():
+            return json_compatible([])
+
+        result = []
+        for uid in uids:
+            item = uuidToObject(uid)
+            links_info = item.restrictedTraverse("@@delete_confirmation_info")
```

Comment:
@ericof @jensens just FYI: we are about to call another browser view in plone.restapi. You know, because: Plone.

Reply:
But don't we better then use `queryMultiAdapter` which is much faster? Do we really need `restrictedTraverse`?